middleware/authentication.go

```diff
@@ -24,7 +24,7 @@ func MakeHTTPBasicAuthenticationMW(passwordToMatch string, logger cs.Logger) fun
 
 			if authorizationHeader == "" {
 				logger.Log("Authorization Error", "Missing Authorization header")
-				httpErrorHandler(context.TODO(), http.StatusForbidden, fmt.Errorf("Missing Authorization header"), w)
+				httpErrorHandler(context.TODO(), http.StatusForbidden, fmt.Errorf("missingAuthorizationheader"), w)
```

Comment:
When message is static, fmt.Errorf(...) can be replaced by errors.New(...)

Reply:
Good remarks. Done